e3sm_diags/driver/utils/climo.py

```diff
@@ -45,6 +45,10 @@ def climo(var, season):
     v = var.asma()
 
     # Compute climatology
+    # FIXME: season can be "ANN" and there is no conditional for "ANN", which
+    # breaks this function. The old Dataset class would call this function
+    # and drivers would do a try and except that captures Exception (all
+    # exceptions), which produces a silent error.
     if season == "ANNUALCYCLE":
         cycle = [
             "01",
```

Comment:
Just a note that the old `climo()` function breaks when `season == "ANN"` because there is not conditional statement to set `cycle`.

Reply:
The comment above is incorrect. There is a conditional for "ANN".